README.md

```diff
@@ -204,12 +203,6 @@ Deploy the application to Minikube using the Marblerun.
     tools/check_manifest.sh tools/manifest.json
     ```
 
-    * If you're running with a custom domain
-
-    ```
-    tools/check_manifest.sh /tmp/manifest.json
-    ```
-
 
 1. Use the app!
```

Comment:
Maybe add some information here about the certificate warning one will receive, since we no longer manually add the certificate to the browser